changelog.txt

```diff
@@ -4,8 +4,6 @@ Changelog
 
 The **synced_collections** package follows `semantic versioning <https://semver.org/>`_.
 
-Version 0
```

Comment:
If we aren't keeping this, we should change the next level to be a `=====` heading instead of `-----`. It matters when this is included into the docs.